### PR TITLE
Story desktop GPU memory optimization so videos play smoothly.

### LINKED ISSUE #23184: Video playback issues on story 3 panels desktop experience.
Video playback fails for some hardware on the story 3 panels desktop experience.

### PR DESCRIPTION
**Pull request by gmajoulet**
The GPU memory used by our desktop three panels experience was so high that on some computers, the hardware-accelerated video decoding was stuttering, resulting on videos not playing.
(Hardware accelerated video decoding uses GPU, not CPU, which is why videos were playing smoothly when we were disabling it).

This fix gets all videos to play as smooth as expected in this desktop three panels layout, but as a nice side effect gets the overall experience much much faster.
Removing this `box-shadow` has no visible effect, since we removed the background images.

One line fix but really interesting performance improvements:

| Device  | Max GPU memory / Min FPS before | Max GPU memory / Min FPS after | Improvement |
| :-------------: | :-------------: | :-------------: | :-------------: |
| Dell workstation (linux)  | ~73mb / 41 FPS | 13mb / 56 FPS | 5.6x less GPU memory used |
| MacBook pro 2017 13i  | ~280mb / 17 FPS | ~60mb / 42 FPS | 4.7x less GPU memory used |

Fixes #23184

### CODE REVIEW

**Comment by gmajoulet** [comment]
`border-radius` is just some cleanup, it all came from the `box-shadow`

**Comment by newmuis** [comment]
FYI @hongwei1990 since we added (or, drastically increased) this when we removed the background images

File: extensions/amp-story/1.0/amp-story-desktop-panels.css [hunk 1]
```diff
@@ -89,7 +89,6 @@ amp-story[standalone].i-amphtml-story-desktop-panels {
   transform: scale(0.9) translateX(calc(250% + 192px)) translateY(0%) !important;
   opacity: 0.05 !important;
   transform-origin: left !important;
-  border-radius: 0 !important;
```
Comment (newmuis) [inline extensions/amp-story/1.0/amp-story-desktop-panels.css]: Do we actually want to clean this up?  Won't this mean publishers can (perhaps accidentally) add a border-radius to pages?

File: extensions/amp-story/1.0/amp-story-desktop-panels.css [hunk 2]
```diff
@@ -186,7 +185,3 @@ amp-story[standalone].i-amphtml-story-desktop-panels {
   min-width: 320px !important;
   min-height: 533px !important;
 }
-
-.i-amphtml-story-desktop-panels > amp-story-page {
-  box-shadow: 0 50px 200px rgba(0, 0, 0, 0.7) !important
```
Comment (newmuis) [inline extensions/amp-story/1.0/amp-story-desktop-panels.css]: Do we want to enforce this is `none`?

File: extensions/amp-story/1.0/amp-story-desktop-panels.css (hunk 1 above)
Comment (gmajoulet) [reply extensions/amp-story/1.0/amp-story-desktop-panels.css]: Done

File: extensions/amp-story/1.0/amp-story-desktop-panels.css (hunk 2 above)
Comment (gmajoulet) [reply extensions/amp-story/1.0/amp-story-desktop-panels.css]: Done, but to clarify this, having a `box-shadow` is fine. The value here is super high (200px!), making it overlap other pages that have an opacity themselves. They overlap each other, animating over several layers of opacity... The browser had to render that every 16ms :(

**Pull request merged**